src/overlays/actors/ovl_En_Owl/z_en_owl.h

```diff
@@ -4,12 +4,65 @@
 #include "ultra64.h"
 #include "global.h"
 
-struct EnOwl;
+typedef enum {
+    OWL_DEFAULT,
+    OWL_OUTSIDE_KOKIRI,
+    OWL_HYRULE_CASTLE,
+    OWL_KAKARIKO,
+    OWL_HYLIA_GERUDO,
+    OWL_LAKE_HYLIA,
+    OWL_ZORA_RIVER,
+    OWL_HYLIA_SHORTCUT,
+    OWL_DEATH_MOUNTAIN,
+    OWL_DEATH_MOUNTAIN2,
+    OWL_DESSERT_COLOSSUS,
+    OWL_LOST_WOODS_PRESARIA,
+    OWL_LOST_WOODS_POSTSARIA
+} EnOwlType;
 
-typedef struct EnOwl {
+typedef enum {
+    OWL_REPEAT,
+    OWL_OK,
+} EnOwlMessageChoice;
+
+typedef struct EnOwl EnOwl;
+
+typedef void (*EnOwlActionFunc)(EnOwl* this, GlobalContext* globalCtx);
+typedef void (*OwlFunc)(EnOwl* this);
+
+typedef struct EnOwl{
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x2C8];
-} EnOwl; // size = 0x0414
+    /* 0x014C */ ColliderCylinder collider;
+    /* 0x0198 */ SkelAnime skelAnime;
+    /* 0x01DC */ Vec3s drawTbl[0x15];
+    /* 0x025A */ Vec3s transitionTbl[0x15];
+    /* 0x02D8 */ SkelAnime skelAnime2;
+    /* 0x031C */ Vec3s drawTbl2[0x10];
+    /* 0x037C */ Vec3s transitionTbl2[0x10];
```

Comment:
the skelanime tables are gonna need the standard names `limbDrawTable` and `transitionDrawTable` until we do a mass renaming